Assets/Scripts/PlayerController.cs

```diff
@@ -29,11 +41,12 @@ void OnMove(InputValue value)
         Debug.Log(moveInput);
     }
 
-    void OnJump (InputValue value)
+    void OnJump(InputValue value)
     {
-        if (value.isPressed)
+        if (value.isPressed && jumpCount < maxJumpCount)
         {
-            rb.velocity += new Vector2(0f, jumpForce);
+            rb.velocity = new Vector2(rb.velocity.y, jumpForce);
+            jumpCount++;
```

Comment:
Currently, the `jumpCount` variable stays as "1", was it supposed to get back to 0 when the player touches the ground?

Reply:
It should reset to zero inside GroundCheck() method you added, when the player touches the ground, and only if the velocity is below or equal to zero. It didn't work befrore I added the velocity check because at the very first frames of the jumping, due to the vertical size of the ground collider, it would still be colliding with the ground and give the player an extra jump. 

As soon as the Jump() method is executed, the velocity.y is set to the jumpForce variable, so we could use that to differentiate between the grounded and jumping states.